### PR TITLE
feat: Send users confirmation when application submitted

### PR DESCRIPTION
**Pull request by DafyddLlyr**
## What does this PR do?
- Sends applicant a "Confirmation" email when their session is flagged with a `submitted_at` timestamp
- Add a bit more clarity around templates and permissions - comments most welcome on the language and naming here

## To test
- Submit an application, check you get an email 📬

### CODE REVIEW

**Comment by DafyddLlyr** [comment]
[Always send a confirmation email to the user when the application has been submitted.](https://trello.com/c/KeCs6vcw/2279-always-send-a-confirmation-email-to-the-user-when-the-application-has-been-submitted)

File: hasura.planx.uk/metadata/tables.yaml [hunk 1]
```diff
@@ -202,8 +202,6 @@
                 _eq: x-hasura-lowcal-session-id
             - email:
                 _eq: x-hasura-lowcal-email
-            - submitted_at:
-                _is_null: true
```
Comment (DafyddLlyr) [inline hasura.planx.uk/metadata/tables.yaml:206]: This feature requires that public users can select submitted sessions. This isn't actually an issue as queries for all sessions (e.g. resume) already have appropriate "WHERE" clauses.

**Review by jessicamcinchak — APPROVED** [review]
Works like a charm, nice to see it come together so quick :racehorse: 
Screenshot from 2023-02-17 14-04-24

Is the plan here to wait to merge/deploy until the Notify template is reviewed by the content team?

**Comment by DafyddLlyr** [comment]
In terms of moving this forward, the plan is as follows - 

- Toggle this feature off (just an `if (false)` or something similar
- Get content written
- Toggle on

I'll add a toggle to this feature now that it's been tested by devs and code reviewed 👍

**Comment by DafyddLlyr** [comment]
Next steps - wait for non-dummy content from Alice, update the template on GovNotify, then merge this 👍

**Pull request merged**